src/api.ts

```diff
@@ -1,11 +1,37 @@
 import express from 'express';
 import { parseQuery, resize, sha256 } from './utils';
-import { set, get, streamToBuffer } from './aws';
+import { set, get, streamToBuffer, remove } from './aws';
 import resolvers from './resolvers';
 import constants from './constants.json';
 
 const router = express.Router();
 
+router.post('/:type/:id/delete', async (req, res) => {
+  const { type, id } = req.params;
+  const { sizes } = req.body;
```

Comment:
The issue with this is that we wont be able to really clear an image from the cache, we would need to populate the different size from the UI, but some might be unknown. Instead of this I recommend make it in a way that when we store image under a folder with a name that use the key of the base image, and when we delete an image cache we just delete everything on the folder with that key. This route can just be a GET request (make it easy to clear cache manually).

Reply:
Okay good idea. Will do the change like this.